kodilla-jdbc/src/test/java/com/kodilla/jdbc/StoredProcTestSuite.java

```diff
@@ -0,0 +1,57 @@
+package com.kodilla.jdbc;
+
+import org.junit.Test;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.junit.Assert.assertEquals;
+
+public class StoredProcTestSuite {
+
+    @Test
+    public void testUpdateVipLevels() throws SQLException {
+        //Given
+        DbManager dbManager = DbManager.getInstance();
+        String sqlUpdate = "UPDATE READERS SET VIP_LEVEL=\"Not set\"";
+        Statement statement = dbManager.getConnection().createStatement();
+        statement.executeUpdate(sqlUpdate);
+
+        //When
+        String sqlProcedureCall = "CALL UpdateVipLevels()";
+        statement.execute(sqlProcedureCall);
+
+        //Then
+        String sqlCheckTable = "SELECT COUNT(*) AS HOW_MANY FROM READERS WHERE VIP_LEVEL=\"Not set\"";
+        ResultSet rs = statement.executeQuery(sqlCheckTable);
+        int howMany = -1;
```

Comment:
czemu -1?

Reply:
To są pytania do twórców kursu bo to oni tak zrobili :) Ten pierwszy test (testUpdateVipLevels()) jest treścią kursu, drugi (testUpdateBestsellers()) to robione na jego wzór zadanie). Sam się zastanawiałem czemu dali -1 i strzelam, że chcieli pokazać, że w pętli poniżej zachodzi zmiana z -1 na 0.